ha/core/event_analyzer/filter/filter.py

```diff
@@ -41,17 +48,28 @@ def filter_event(self, msg: str) -> bool:
         """
         pass
 
+    @staticmethod
+    def is_iem_msg(msg: dict) -> bool:
```

Comment:
is_iem_msg instade can we use it 
get_msg_type
to support more format and can be use by alert if needed

Reply:
Agree

Reply:
Done.